jdaviz/core/template_mixin.py

```diff
@@ -1463,6 +1462,22 @@ def _sort_by_icon(items_dict):
 
         self._apply_default_selection()
 
+    def set_wcs_only_filter(self, wcs_only):
```

Comment:
I'm still not convinced this belongs here vs in orientation plugin, but can see arguments both ways.  `set` might be confusing though... how about `toggle` or separate `add`/`remove` methods and then do the check externally?

Either way, this code is going to be a bit difficult to parse in the future, so could use a docstring for future us.